specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json

```diff
@@ -53,7 +53,7 @@
         "tags": [
           "SKU"
         ],
-        "operationId": "SKUs_List",
+        "operationId": "Skus_List",
         "description": "Lists the available SKUs supported by Microsoft.Storage for given subscription.",
         "x-ms-examples": {
           "SKUList": { "$ref": "./examples/SKUList.json" }
```

Comment:
SkuList.

To be consistent.

Reply:
I renamed the `tags` to `Skus` is this enough? I would prefer the `s`, for consistency with `StorageAccounts`